libraries/AP_Scripting/examples/plane-doublets.lua

```diff
@@ -65,7 +61,7 @@ function doublet()
             pre_doublet_mode = vehicle:get_mode()
             -- are we doing a doublet on elevator or rudder? set the other controls to trim
             local doublet_choice_pwm = rc:get_pwm(DOUBLET_CHOICE_CHANNEL)
-            local trim_funcs = {}
+            local trim_funcs
```

Comment:
libraries/AP_Scripting/examples/plane-doublets.lua:64:19: value assigned to variable trim_funcs is unused